scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -862,8 +863,15 @@ export function AuthoringDirective(
             };
 
             $scope.sendToNextStage = function() {
-                var stageIndex, stageList = desks.deskStages[desks.activeDeskId];
-                var selectedStage, selectedDesk = desks.deskLookup[desks.activeDeskId];
+                var currentDeskId = desks.getCurrentDeskId();
+
+                if (_.isNil(currentDeskId)) {
+                    notify.error(gettext('Failed to send to next stage.'));
```

Comment:
When is `currentDeskId` null/undefined? Is it a valid use case or an error? If it's an error, add a `logger.warn` call so the error gets sent to sentry and we can find and fix it.

Reply:
currentDeskId could be null if the user is working on his personal space so I don't think it is an error

Reply:
Can you add a reason to the error message so it's clear to the user why the action doesn't work?